packages/ui/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+    "compilerOptions": {
+        // TODO: Remove this once we have migrated to TS
+        "allowJs": true,
```

Comment:
The allowJs option in a tsconfig.json file allows the TypeScript compiler to include and process .js (JavaScript) files alongside .ts (TypeScript) files. This is useful when: We're migrating a JavaScript project to TypeScript gradually.